src/pages/publication/new_publication/LoadPublication.tsx

```diff
@@ -50,7 +51,7 @@ const LoadPublication: React.FC<LoadPublicationProps> = ({ expanded, onChange, u
       ariaControls="publication-method-file">
       {uppy ? (
         <>
-          <UppyDashboard uppy={uppy} />
+          <UppyDashboard uppy={uppy} shouldAllowMultipleFiles={shouldAllowMultipleFiles} />
```

Comment:
jo mer jeg ser på denne propen, jo mer skulle jeg ønsket at den het `allowMultipleFiles` fordi `should` betyr jo ikke at den faktisk gjør det 😛

Reply:
Ønsket å bruke prefix-konvensjonen mange anbafaler på boolske variablenavn  (f.ex https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig )

Reply:
fair enough 👍